Makefile

```diff
@@ -21,7 +21,11 @@ INSTALL ?= install
 ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 # Base path used to install.
-DESTDIR ?= /usr/local
+PREFIX        ?= /usr/local
+SYSCONFDIR    ?= /etc
```

Comment:
Doesn't seem used

Reply:
is used by scripts and downstream makefiles that we use for local and release installs.